src/main/java/be/seeseemelk/mockbukkit/block/BlockMock.java

```diff
@@ -344,15 +344,18 @@ public PistonMoveReaction getPistonMoveReaction()
 	@Override
 	public boolean breakNaturally()
 	{
```

Comment:
I'm seeing in craftblock that it does actually drop items. Does it call the `BlockDropItemEvent` in that case? But also would the item drop need to be mocked?

Reply:
The PR doesn't simulate either the `BlockDropItemEvent` event (missing `Block#getDrop`) and the item drop, and item drops would also have to be mocked (if the player can pick up items from the world).

Reply:
Adding that functionality wouldn't be too hard I think. It would require an `ItemMock` entity which gets added to the list of entities in a world, and implement some simple methods. However, it would probably be best to implement this in a separate PR.